opentelemetry-otlp/allowed-external-types.toml

```diff
@@ -0,0 +1,22 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
(sorry to nit pick) - Not related to this PR, we don't include copyright/license headers in individual files, we have the `LICENSE` file in each crate, and also license information in `crates.toml`. I noticed that the header is present in other `allowed-external-types.toml`'s too, which seems like something we should address in a separate cleanup :)

Reply:
+1